source/library/Argo/Schema/Schema.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveLift #-}
+
+module Argo.Schema.Schema where
+
+import qualified Argo.Json.Array as Array
+import qualified Argo.Json.Boolean as Boolean
+import qualified Argo.Json.Member as Member
+import qualified Argo.Json.Name as Name
+import qualified Argo.Json.Object as Object
+import qualified Argo.Json.String as String
+import qualified Argo.Json.Value as Value
+import qualified Argo.Vendor.DeepSeq as DeepSeq
+import qualified Argo.Vendor.TemplateHaskell as TH
+import qualified Data.Text as Text
+import qualified GHC.Generics as Generics
+
+-- | A JSON Schema.
+-- <https://json-schema.org/draft/2020-12/json-schema-core.html>
+newtype Schema
+    = Schema Value.Value
```

Comment:
This is not a great representation of a JSON schema for a number of reasons. However it is extremely flexible, which feels important considering how much the spec changes from draft to draft. 

A better representation would be more structured and only convert into an actual JSON value at the last possible moment.